maksim_khier/README.md

```diff
@@ -0,0 +1,75 @@
+Before running the application, create databases "wiredcraft" and "wiredcraft_test" in local
+mysql
+
+To get dependencies:
+Run 'go get'
+
+To build the exectuable:
+Run 'go build -o wiredcraft_test'
+
+To run the executable:
+Run './wiredcraft_test'
+The server will be running on localhost:8080
+
+To test:
+Run 'go test'
+
+**************************************************
+
+API endpoints:
+
+All API endpoints are under /api namespace.
+
+User model:
+```
+{
+  name string
+  dob string
+  address string
+  description string
+  createdAt string
+}
+```
+
+**Create a user**
+Endpoint: ```POST /user```
+Send payload as json string in body:
+```
+{
+  "name": "Boris",
+  "dob": "1950-01-01T12:00:00Z",
+  "address": "London",
+  "description": "UK foreign minister"
+}
+```
+
+###########################################################################
```

Comment:
- Provide a complete user auth (authentication/authorization/etc) strategy, such as OAuth.
I can not find this related API.

Reply:
added in new commit